app/src/main/java/com/sma2/sma2/DataAccess/ExerciseDataService.java

```diff
@@ -0,0 +1,65 @@
+package com.sma2.sma2.DataAccess;
+
+import android.content.Context;
+
+import com.sma2.sma2.ExerciseLogic.Exercise;
+
+import org.greenrobot.greendao.database.Database;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExerciseDataService {
+    private Context invocationcontext;
+    public ExerciseDataService(Context context){
+        this.invocationcontext = context;
+    }
+
+    public void saveExercise(Exercise exercise){
+        ExerciseDA tmp = new ExerciseDA(exercise);
+        DaoMaster.DevOpenHelper helper = new DaoMaster.DevOpenHelper(invocationcontext, "apkinsondb");
+        Database db = helper.getWritableDb();
+        DaoSession session = new DaoMaster(db).newSession();
+        session.getExerciseDADao().save(tmp);//todo change the real thing
+        db.close();
+    }
+
+    public void updateExercise(ExerciseDA exercise){
+        DaoMaster.DevOpenHelper helper = new DaoMaster.DevOpenHelper(invocationcontext, "apkinsondb");
+        Database db = helper.getWritableDb();
+        DaoSession session = new DaoMaster(db).newSession();
+        session.getExerciseDADao().update(exercise);
+        db.close();
+    }
+
+    public void deleteExercise(ExerciseDA exercise){
+        DaoMaster.DevOpenHelper helper = new DaoMaster.DevOpenHelper(invocationcontext, "apkinsondb");
+        Database db = helper.getWritableDb();
+        DaoSession session = new DaoMaster(db).newSession();
+        session.getExerciseDADao().delete(exercise);
+        db.close();
+    }
+    public Exercise getExercise(){
```

Comment:
Shouldn't this function at least get a primary key to specify which excerse I want?